_data/utilities.yml

```diff
@@ -469,3 +469,14 @@ websites:
   bch: false
   btc: false
   othercrypto: false
+- name: igiftcards
+  url: https://igiftcards.nl
+  img: igiftcards.png
+  bch: Yes
+  btc: Yes
+  othercrypto: Yes
+  region: eu
+  country: NL
+  doc: Note: several other country domains: fr/de/at/it/es exist as well
```

Comment:
Needs quotes here I think

Reply:
I shall work on the automation to identify when needed and add the quotes.

If that seems to be more complicated than one would hope, I can just add the quotes every time, wouldn’t be any harm